07 Data Library/05 Futures/05 Reference Tables.php

```diff
@@ -34,16 +37,20 @@
         <tbody>
         <?php
         foreach ($decoded['fields'] as $field)
-        { ?>
-            <tr>
-                <td class="futures-type">
-                    <p><?=htmlentities($field['field-description'])?></p><br/>
-                </td>
-                <td>
-                    <code><?php echo "Futures.{$typeName}.{$field['field-name']}"; ?></code>
-                </td>
-            </tr>
-  <?php } ?>
+        { 
+            if (!in_array($field['field-description'], $ice_data))
```

Comment:
To avoid indentation try and write things in the positive - continue sense; 
`"if in array, continue "`
then it'll also be a more localized change.